.travis.yml

```diff
@@ -117,31 +107,19 @@ matrix:
       - RUSTFLAGS="-D warnings" cargo build --verbose --target=$TARGET_32
       - RUSTFLAGS="-D warnings" cargo test --verbose --target=$TARGET_32
    - name: "stable Android"
-     language: android
-     dist: trusty
+     language: rust
+     rust: stable
      env:
        NDK_VER_OLD=r13b
        NDK_VER=r21
-       CMAKE_VER=3.6.4111459
-     android:
-       components:
-        - build-tools-26.0.1
-        # Minimum API level supported
-        - android-21
+       NDK_URL=https://dl.google.com/android/repository/android-ndk-%s-linux-x86_64.zip
```

Comment:
I think it's better to leave this in the `script` section, so it doesn't show up in the build configuration https://travis-ci.com/github/cloudflare/quiche/jobs/361909618 which is not very useful.

Reply:
Done.

Reply:
And rebased.